db/seeds.rb

```diff
@@ -1,109 +1,144 @@
 USERS_TO_CREATE = 20
-USERS_TO_CREATE.times do
-
-	User.create company:					Faker::Company.name,
-							email:						Faker::Internet.email,
-							password:	        Faker::Internet.password,
-							employees:				rand(8),
-							website:					Faker::Internet.url
+NUMBER = 0
 
+USERS_TO_CREATE.times do
+  NUMBER += 1
+  User.create(company: Faker::Company.name + NUMBER.to_s,
+              email: NUMBER.to_s + Faker::Internet.email,
+              password: Faker::Internet.password,
+              employees: rand(8),
+              website: Faker::Internet.url + NUMBER.to_s)
 end
-# works
 
-User.create company:					"Admin Man",
-						email:						"admin@admin.com",
-						password:	        "123456",
-						employees:				rand(8),
-						website:					Faker::Internet.url,
-						admin:						true
+CREATE_ONE_ADMIN = 1
+CREATE_ONE_ADMIN.times do
+  User.create(company: 'Admin Man',
+              email: 'admin@admin.com',
+              password: '123456',
+              employees: rand(8),
+              website: Faker::Internet.url,
+              admin: true)
+end
 
-User.create company:					"Dev Man",
-						email:						"dev@dev.com",
-						password:	        "123456",
-						employees:				rand(8),
-						website:					Faker::Internet.url,
-						admin:						false
+CREATE_ONE_DEV = 1
+CREATE_ONE_DEV.times do
+  User.create(company: 'Dev Man',
+              email: 'dev@dev.com',
+              password: '123456',
+              employees: rand(8),
+              website: Faker::Internet.url,
+              admin: false)
+end
 
-business = ["Running", "Closed", "Updating"]
+BUSINESS_STATUSES = %w(Running Closed Updating).freeze
 5.times do
-    Location.create name:           Faker::Company.name,
-                  	address:        Faker::Address.street_address,
-                    city:           Faker::Address.city,
-                    postal_code:    Faker::Address.zip_code,
-                    run_status:     business.sample
+  NUMBER += 1
+  Location.create(name:           Faker::Company.name + NUMBER.to_s,
+                  address:        Faker::Address.street_address + NUMBER.to_s,
+                  city:           Faker::Address.city + NUMBER.to_s,
+                  postal_code:    Faker::Address.zip_code + NUMBER.to_s,
+                  run_status:     BUSINESS_STATUSES.sample)
 end
 
 20.times do
-    Machine.create location_id:     1+rand(5)
+  Machine.create location_id:     1 + rand(5)
+end
+
+LIST_OF_STATUSES = %w(under_review
```

Comment:
This can be deleted. Statuses are no longer in use. We are using AASM_state from now on. This table should be deleted.